index.html

```diff
@@ -1,11 +1,58 @@
+	<h1>Nick Chemsak's Portfolio</h1>
+	<h2>Pursuing Creative Passions</h2>
+
+	<!-- 3. Photo -->
+	<img src="images/biopic3.jpg" alt="Photo of Nick">
+
+	<!-- 4. Article element containing your bio -->
+	<article>
+		<p>Nick grew up in the central state of Hesse <span class="wordpop">Germany</span>, just north of Frankfurt.  While visiting Berlin on November 9th 1989, he experienced the fall of the <span class="wordpop">Berlin Wall</span> first-hand.  </p>
+		<p>In 2003, he graduted with a Bachelor of Science in the Music Recording Industry program at <span class="wordpop">Middle Tennessee State University</span>, and went on to produce and engineer songs for many songwriters and bands.  He also wrote, produced, and engineered original music for 2 films which were entered into <span class="wordpop">film festivals</span> in Nashville TN and Atlanta GA.  </p>
+		<p>Nick has always had a passion for technology and how it can push the boudaries of knowledge and human experience.  He has coded as a hobby for the past 8 years while working in the IT field at a Credit Union.  Just recently, he has enrolled in the <span class="wordpop">Nashville Software School</span> to persue this passion professionally.  In his free time, he is an avid painter, and just recently opened his first online store, <a href="http://www.ephemeralwave.com" target="_blank"> Ephemeral Wave</a>.  He enjoys hiking, playing football, and spending time with his young son and wife.
+	</article>
+</div>
+
+	<!-- 5. Article element with links to resources or sites you like -->
+	<article class="divide">
+		<h3>Resources</h3>
+			<ul>
+		 		<li><a href="https://daneden.github.io/animate.css/" target="_blank">CSS animation</a></li>
+		 		<li><a href="http://validator.w3.org/" target="_blank">HTML validator</a></li>
+		 		<li><a href="http://www.programmableweb.com/apis" target="_blank">APIs</a></li>
+		 		<li><a href="http://www.teamtreehouse.com" target="_blank">Team Treehouse</a></li>
+		 		<li><a href="https://scotch.io/bar-talk/best-of-sublime-text-3-features-plugins-and-settings" target="_blank">Best of Sublime Text</a></li>
+		 		<li><a href="https://necolas.github.io/normalize.css/" target="_blank">Normalize CSS</a></li>
+	 		</ul>
+	</article>
+
+
+<!-- 	6. Article element with two sections for future projects (placeholders) -->
+	<article class="divide">
+		<h3>Future Projects</h3>
+		<div>Placeholder 1</div>
+		<div>Placeholder 2</div>
+	</article>
+
+	<!-- 7. Footer element with email and professional social media links; Github, twitter, LinkedIn, etc. -->
+	<footer class="divide"><h3>Contact </h3><a href= "mailto:nchemsak@gmail.com">nchemsak@gmail.com</a> <a href="https://github.com/nchemsak" target="_blank"> Github </a><a href="https://www.linkedin.com/in/nick-chemsak-3084b09" target="_blank"> LinkedIn </a> <a href="http://www.codepen.io/nchemsak" target="_blank">CodePen</a></footer>
+  
```

Comment:
Be very careful about leaving trailing whitespace in your code. It will inevitably lead to merge conflicts. You can see whitespace by adding a plugin called Trailing Spaces to your sublime, or you can add this line to your User Preferences and stop them out right. "trim_trailing_white_space_on_save": true